OSXNotifier.go

```diff
@@ -44,5 +46,10 @@ func (n *osxNotifier) Notify(msg *Notification) error {
 		return err
 	}
 	log.Debugf("Received result: %v", string(result))
+	if msg.ClickURL != "" {
+		if string(result) == "Open" {
```

Comment:
This is the heart of the change. We specify our own action, which allows us to get the result of the click here, and then we just open the URL ourselves.